tf_static_publisher/CMakeLists.txt

```diff
@@ -4,11 +4,12 @@ project(tf_static_publisher)
 find_package(catkin REQUIRED COMPONENTS
   geometry_msgs
   roscpp
+  std_srvs
   tf2_ros
 )
 
 catkin_package(
-  CATKIN_DEPENDS geometry_msgs roscpp tf2_ros
+  CATKIN_DEPENDS geometry_msgs roscpp std_srvs tf2_ros
```

Comment:
This is not correct. Packages here are build_export_depends and this package has none as it doesn't provide any libraries or headers.